spec/controllers/container_image_registry_controller_spec.rb

```diff
@@ -18,7 +18,7 @@
     get :show, :id => container_image_registry.id
     expect(response.status).to eq(200)
     expect(response.body).to_not be_empty
-    expect(assigns(:breadcrumbs)).to eq([{:name => "Container Image Registry",
+    expect(assigns(:breadcrumbs)).to eq([{:name => "Container Image Registries",
```

Comment:
> @dkorn let's try to be strict on this. At least split the patch in two (same PR).

@simon3z Devided the PR into 2 commits:
a. Fix Image Registries display_name string 
b. Refactor: container common mixin